### DISCUSSION TITLE
HRR decay

### DISCUSSION THREAD

**Discussion by ShehabGamrah** [firemodels/fds#13592] [Q&A] [answered]
![{C3425534-7C5A-430F-B93A-2E20AD707BEC}](https://github.com/user-attachments/assets/7d94cfc0-f687-4ee4-8b3f-2bbc4a23731b)

I would like to ask within FDS as we are capable to specify the ramp up time of a fire using HRR, how can we specify the decay time as in this curve. Also with regards to the temperature boundary condition of a burner surface. It is normally set at 300C by default. Does changing it affect temperature within the simulation.

**Comment by drjfloyd** [comment, marked as answer]
Please read the User's Guide. It discusses how to define a time dependent heat release rate for a fire.

In terms of temperature the default surface thermal condition in FDS is not 300. Pyrosim may set this temperature, but that is not an FDS default, it is a decision made by Thunderhead Engineering. A thought experiment. If you ran two cases with the surface temperature set to absolute zero (0 K) and 5000 K, would you expect to see any differences?

**Reply by ShehabGamrah** [reply]
Thanks for your answer. regarding the temperature if I use a high-intensity fires like a 30 MW polyurethane fire, would the burner temperature boundary condition drastically alter the overall temperature results, as the high HRR will quickly dominate the local heat transfer?

**Reply by mcgratta** [reply]
I doubt it, but you can easily test this.
